content/codes.md

```diff
@@ -27,6 +27,8 @@ toc: true
 - [rdseed](http://ds.iris.edu/ds/nodes/dmc/forms/rdseed/): Convert seismic data from SEED format to other common format (SAC et al.) |
   [Chinese notes](https://blog.seisman.info/tags/SEED)
 - [sac2mseed](https://github.com/iris-edu/sac2mseed): Convert SAC format to minniSEED format
+- [libmseed](https://github.com/iris-edu/libmseed): A miniSEED library that
```

Comment:
`libmseed` is mainly used to read/write miniseed, so it belongs to format conversion.